src/Microsoft.Identity.Client/Internal/JsonWebToken.cs

```diff
@@ -2,8 +2,11 @@
 // Licensed under the MIT License.
```

Comment:
Can this file pls. be split into 3 different files, with the 3 top level classes?

Reply:
ill make the change